### PR TITLE
Email configurations added for production server

### PR DESCRIPTION
**Pull request by deshraj**

### CODE REVIEW

File: settings/prod.py [hunk 1]
```diff
@@ -35,3 +35,12 @@
 MEDIAFILES_LOCATION = 'media'
 MEDIA_URL = 'http://%s.s3.amazonaws.com/%s/' % (AWS_STORAGE_BUCKET_NAME, MEDIAFILES_LOCATION)
 DEFAULT_FILE_STORAGE = 'custom_storages.MediaStorage'
+
+# Setup Email Bacend related settings
+DEFAULT_FROM_EMAIL = "noreply@cloudcv.org"
```
Comment (dexter1691) [inline settings/prod.py]: Do we have this email?

Comment (deshraj) [inline settings/prod.py]: No, we dont. But, we need to have one.

Comment (deshraj) [inline settings/prod.py]: @dexter1691 can you approve this?

**Comment by dexter1691** [comment]
@deshraj : can you resolve the conflicts

**Pull request merged**